app/src/main/java/com/example/peterchu/watplanner/MainActivity.java

```diff
@@ -37,13 +39,32 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onResponse(Call<CourseResponse>call, Response<CourseResponse> response) {
                 Log.d("MainActivity", response.toString());
                 List<Course> courses = response.body().getData();
-                Log.d("MyActivity", "Number of courses received: " + courses.size());
+                Log.d("MainActivity", "Number of courses received: " + courses.size());
             }
 
             @Override
             public void onFailure(Call<CourseResponse>call, Throwable t) {
                 // Log error here since request failed
-                Log.e("MyActivity", t.toString());
+                Log.e("MainActivity", t.toString());
+            }
+        });
+
+        Call<CourseScheduleResponse> scheduleCall = apiService.getSubjectCourseSchedules(
+                "1175", "ECE", Constants.API_KEY
```

Comment:
Note: sooner or later we'll have to replace the hardcoded 1175 to a dynamically generated string (which should be easy)